cmd/multiFaReduce/testdata/expected.mfa

```diff
@@ -0,0 +1,8 @@
+>one
+AACC
```

Comment:
How can this output file be used downstream? Intuitively I think it might be confusing that after removing the non-variable columns, the remaining variable columns are smashed together, so we don't know their original positions in the original mfa. I would have thought mfas would show the alignments like this:
>one
----AACC--
>two
----AAAA--

Reply:
You make a good point that after reducing an alignment to only the segregating sites, we can not recover the original alignment, or even map the segregating sites back to the original alignment.  I think the downstream analyses will be to calculate statistics on these segregating sites and they will not be brought back to the bigger alignments, but we could consider some way to keep track of things so that more of the original alignment could be recovered.